config/main.py

```diff
@@ -1321,13 +1391,21 @@ def vlan_member(ctx):
 def add_vlan_member(ctx, vid, interface_name, untagged):
     """Add VLAN member"""
     log_info("'vlan member add {} {}' executing...".format(vid, interface_name))
+    if sonic_device_util.is_multi_npu():
+        # Get the namespace based on the member interface given by user.
+        intf_ns = get_intf_namespace(interface_name)
+        if intf_ns is None:
+            ctx.fail("member interface {} is invalid".format(interface_name))
+        elif intf_ns != ctx.obj['namespace']:
+            ctx.fail("member interface {} doesn't exist in namespace {}".format(interface_name, ctx.obj['namespace']))
```

Comment:
this check logic is special, just connect to ctx.obj['namespace'] and check if port is there or not, make it work for single asic too.

Reply:
I updated to use the existing function interface_name_is_valid(), as in this case we already have the details of which config db to connect to.